internal/processor/otel/logs.go

```diff
@@ -120,12 +128,56 @@ func (c *Consumer) convertLogRecord(
 		}
 		event.Span.ID = spanID.HexString()
 	}
-	if attrs := record.Attributes(); attrs.Len() > 0 {
+	attrs := record.Attributes()
```

Comment:
I think we should in addition set the event categorization fields (for all of the events / logs):
* `event.kind` = `event`
* `event.category` = value of the `event.domain` OTel attribute
* `event.type` = `info` (in case of logs / normal events) and `error` in case of errors / crashes, etc

Reply:
Added! :+1: